src/lib/modules/tests/test.js

```diff
@@ -311,19 +311,27 @@ class Test {
   }
 
   require(path) {
-    const prefix = 'Embark/contracts/';
-    if (!path.startsWith(prefix)) {
-      throw new Error(__('Unknown module %s', path));
+    const prefixes = ['Embark/contracts/', 'Embark/EmbarkJS'];
```

Comment:
You seem to never use prefixes, what about deconstructuring it:
```
const [contractsPrefix, embarkJSPrefix] = ['Embark/contracts/', 'Embark/EmbarkJS'];
```

=> prefixes[0] and prefixes[1] are not supper meaningful

Reply:
Smart